ui/src/main/webapp/app/templates/applicationlist.component.html

```diff
@@ -21,7 +21,7 @@
                 Not Started
             </span>
             <span *ngIf="status(app).completed">
-                Done!
+                Done - <a href="file://{{app.outputPath}}/index.html">see the report</a>.
```

Comment:
For the moment, this should probably open a new tab. Otherwise, I think this is fine for now.